dockerproxy/main.go

```diff
@@ -273,9 +276,11 @@ func dockerProxy() http.Handler {
 			}
 		}
 		if !allowed {
-			log.Warnf("Refusing to proxy %s", r.URL)
-			http.Error(w, `{"message":"page not found"}`, http.StatusNotFound)
-			return
+			log.Warnf("Invalid path path=%s agent=%q", r.URL, r.UserAgent())
```

Comment:
Might be good to add an identifiable string to the log message to make it easier to search for.